public/scripts/common/lib/handlebars.mixed.js

```diff
@@ -29,10 +29,53 @@ Displays select tag with options (and optional optgroups)
   * `selected`
 
 **/
+
 Handlebars.registerHelper('select', function(context, options) {
-  var output = `<select id="${options.hash.id}" name="${options.hash.name}" class="${options.hash.class}">`;
-  output += makeOptions(context, options);
-  return output +'</select>';
+  var addOption, addOptions,
+      output = '';
+
+  addOption = function(selected, _options) {
```

Comment:
I copied the bulk of this function from https://github.com/Synbiota/GENtle2-synbiota/blob/develop/public/scripts/common/lib/handlebars.mixed.js which doesn't make use of makeOptions().

I'm happy to do as you requested, just want to check with you first if the above fact changes your request in any way.